Add html-movie-seat-booking

Ant-C-tech opened the pull request:
Task: html-movie-seat-booking

[Demo](https://ant-c-tech.github.io/Cinema-Application/)

The code is submitted in a dedicated feature branch.
Please, review.

Ant-C-tech committed:
Add html-movie-seat-booking

A-Ostrovnyy commented on submissions/Ant-C-tech/html-movie-seat-booking/index.html:
```diff
@@ -0,0 +1,466 @@
+          </ul>
+        </nav>
+
+        <nav class="movieDate">
```
Why you use `<nav>` for this section?

Ant-C-tech replied:
I have some doubts about correct tags for these elements but using this resource http://yoksel.github.io/easy-markup/first-steps/ according your advice I have thought that they have to be section-tags because I consider that for this page of application "controlSection"-element acts a part of "main"-element. I hope that I did it properly.

A-Ostrovnyy commented on submissions/Ant-C-tech/html-movie-seat-booking/index.html:
```diff
@@ -0,0 +1,466 @@
+            <a href="#" class="btn-prevDay">&#9668;</a>
+              <ol class="calendar">
+                <li class="calendar__listItem">
+                  <a href="#" class="calendar__listItemLink calendar__listItemLink">
```
Does this link guide you to another page/part of the current page?

A-Ostrovnyy commented on submissions/Ant-C-tech/html-movie-seat-booking/index.html:
```diff
@@ -0,0 +1,466 @@
+          </ul>
+        </nav>
+
+        <nav class="hall">
```
Same as at line 157

A-Ostrovnyy commented on submissions/Ant-C-tech/html-movie-seat-booking/index.html:
```diff
@@ -0,0 +1,466 @@
+          <p class="movieTime__title">Time</p>
+          <ul class="movieTime__list">
+            <li class="movieTime__listItem  movieTime__listItem-active">
+              <a class="movieTime__listItemLink  movieTime__listItemLink-active" href="#">14:00</a>
```
For date and time preferable to use `<time>` tag [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time)

A-Ostrovnyy commented on submissions/Ant-C-tech/html-movie-seat-booking/index.html:
```diff
@@ -0,0 +1,466 @@
+            <p>From<br>01 March<br>1h 39m</p>
+        </div>
+
+        <div class="mask"></div>
```
Mabey better use pseudoelement for `.posterSection` instead of creating an empty element.

A-Ostrovnyy reviewed (changes requested):
Good work!

Ant-C-tech committed:
Add changes after code-review #1 2021-01-31

Ant-C-tech commented:
Thank you for your review. Your feedback will help me to make my code better and I think that mentor's reviews is the most important part of the course. Apart from your notices I made some work with "badge"-element (added link to IMBd and time-tag for premiere day). I hope it will not surprise you.

A-Ostrovnyy commented on submissions/Ant-C-tech/html-movie-seat-booking/index.html:
```diff
@@ -0,0 +1,477 @@
+        <section class="movieTime">
+          <p class="movieTime__title">Time</p>
+          <ul class="movieTime__list">
+            <li class="movieTime__listItem  movieTime__listItem-active">
+              <a class="movieTime__listItemLink  movieTime__listItemLink-active" href="#">
+                <time datetime="14:00">14:00</time>
+              </a>
+            </li>
+            <li class="movieTime__listItem">
+              <a class="movieTime__listItemLink" href="#">
+                <time datetime="17:00">17:00</time>
+              </a>
+            </li>
+            <li class="movieTime__listItem">
+              <a class="movieTime__listItemLink" href="#">
+                <time datetime="20:00">20:00</time>
+              </a>
+            </li>
+            <li class="movieTime__listItem  movieTime__listItem-disabled">
+              <a class="movieTime__listItemLink movieTime__listItemLink-disabled" href="#">
+                <time datetime="23:00">23:00</time>
+              </a>
+            </li>
```
As for me, it looks like a group of radio buttons: group radio buttons allow you to choose only one item from the group. It's native behaviour in browser no need to add js for it.

A-Ostrovnyy commented on submissions/Ant-C-tech/html-movie-seat-booking/index.html:
```diff
@@ -0,0 +1,477 @@
+          <div class="flexRow">
+            <a href="#" class="btn-prevDay">&#9668;</a>
+              <ol class="calendar">
+                <li class="calendar__listItem">
```
Same as on lines 198-217. What do you think about that way?

A-Ostrovnyy reviewed (changes requested):
Good work!
See my comments below.

A-Ostrovnyy commented on submissions/Ant-C-tech/html-movie-seat-booking/index.html:
```diff
@@ -0,0 +1,477 @@
+
+          <div class="hall__row">
+            <label class="seat">
+              <input class="seat__input visually-hidden" type="checkbox" disabled>
```
Add styles on `:hover`. Also, add attribute value.

Ant-C-tech committed:
Add changes after code review #2 2021-02-02

Ant-C-tech commented:
Thank you for your review!
You totally right: elements for choosing properties of movie have to be radio buttons. I don't know why I did not understand it before - that is why code review is very important - a lot of things I just can't see. So, I made all these changes and added ":hover" and "value" for seats.
Also, if you do not mind, I added custom ":focus" for all interactive elements.
Thank you for your work.

A-Ostrovnyy reviewed (approved):
Well done!
